doc/user_guide/dynamo0p3.rst

```diff
@@ -1343,6 +1378,15 @@ checks (when generating the PSy layer) that any kernels which read
 operator values do not do so beyond the level-1 halo. If any such
 accesses are found then PSyclone aborts.
 
+.. _lfric-array-sizes:
+
+Array sizes
```

Comment:
I think this should be included in the same example where the fourth and fifth entries for fields and operators are described.

Also, I may be missing something here but why `NRANKS*n`? I think that is confusing as it mixes the notation used for a field vector with something different. Can we not just have "n"? I suppose we could have `GH_ARRAY*n` by analogy with a field vector and do away with this extra entry altogether. In fact, maybe it should be `GH_SCALAR*n` as that really does then line up with the way we do field vectors?

Reply:
For posterity: the reason behind not using `GH_ARRAY*n` (or `GH_SCALAR*n`) is because `n` here does not represent the number of rows in a vector, but the number of dimensions in an array (so `n=1` is a vector, `n=2` is a 2d-array, etc)